core/src/main/resources/jenkins/management/Messages.properties

```
@@ -56,5 +56,7 @@ NodesLink.Description=Add, remove, control and monitor the various nodes that Je
 ShutdownLink.DisplayName_prepare=Prepare for Shutdown
 ShutdownLink.DisplayName_cancel=Cancel Shutdown
 ShutdownLink.Description=Stops executing new builds, so that the system can be eventually shut down safely.
+ShutdownLink.ShutDownReason_title=Shut down reason
+ShutdownLink.ShutDownReason_update=Update reason text
```

Seems unneeded to have text in the wording
```suggestion
ShutdownLink.ShutDownReason_update=Update reason
```